src/pages/ProductListPage/components/ProductItem.tsx

```diff
@@ -12,20 +15,29 @@ type productType = {
 
 interface Props extends HtmlHTMLAttributes<HTMLDivElement> {
   item: productType;
-  isSelected: boolean;
-  onSelect: () => void;
+  cartItem: CartItemType | undefined;
 }
 
-const ProductItem = ({ item, isSelected, onSelect }: Props) => {
+const ProductItem = ({ item, cartItem }: Props) => {
+  const { addCartItem } = useAddCartItem();
+  const { deleteCartItem } = useDeleteCartItem();
+
   const { name, price, imageUrl } = item;
+  const isInCart = cartItem ? true : false;
```

Comment:
```suggestion
  const isInCart = !!cartItem;
```

이렇게 표현해볼 수도 있을 것 같아요!